src/js-api-specification.md

```diff
@@ -71,6 +73,52 @@ Live
 
 ### Navigation <a name="navigation"></a>
 
+#### setBackAction Function <a name="setBackAction"></a>
+
+`nhsapp.navigation.setBackAction(backAction: function)`
+
+##### Description
+
+Set the action that will be executed when a user presses the native back button on thier device.
+
+##### Parameters
+
+| Parameters              |                                                                                                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| backAction                 | **Type:** Function *Required*<br>The function to be excuted when the native back is pressed.            
+
+##### Example Usage
+
+```js
+let backAction = () => alert('Hello World!')
```

Comment:
Best to change this to a real life example - e.g. calling into the NHS goToPage method